data.sql

```diff
@@ -1,5 +1,10 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (animal_name, date_of_birth, escape_attempts,
+neutered, weight_kg) 
+VALUES 
+('Agumon', '2020-02-03', 0, FALSE,10.23),
+('Gabumon', '2018-11-15', 2, TRUE,8),
+('Pikachu', '2021-01-07', 1, FALSE,15.04),
+('Devimon', '2017-05-12', 5, TRUE,11)
```

Comment:
- Hey, let's not forget that the datatype for the `weight_kg` is a decimal, so we should send the correct value to it. update to `11.0`